Makefile

```diff
@@ -4,13 +4,13 @@ SHELL := /bin/bash
 # Setup
 ################################
 
-setup: setup-python setup-poetry install-deps setup-pre-commit setup-node setup-fern
+setup: setup-python setup-poetry install-deps setup-pre-commit setup-fern
```

Comment:
@dvargas92495 for context, including `setup-node` produced errors for both Kira and I, so we removed for now. Kira, if you still have the console output handy with that error, could you show here?

Reply:
I don't have the console output for that anymore, sorry!

Reply:
do we remember what the general vibe of the console output was?

Reply:
Something to do with nvm I think?